tegel/src/components/link/sdds-link.scss

```
@@ -1,8 +1,13 @@
+@import '../../mixins/box-sizing';
+
 a {
+  @include sdds-box-sizing;
+
   cursor: pointer;
   outline: none;
-  color: var(--sdds-link);
 
+  &,
```

```suggestion
  color: var(--sdds-link);
  
  
```

To me, it's more readable to have a "default color". But this is just a matter of taste, so whatever you feel it the best choice, go with that one!

---

I agree, I think I did copy it from the native one here, and I will do corrections on both as per your suggestion, it is more straightforward.